core/src/it/scala/org/ensime/fixture/AnalyzerFixture.scala

```diff
@@ -31,6 +32,7 @@ trait IsolatedAnalyzerFixture
     withVFS { implicit vfs =>
       withTestKit { testkit =>
         import testkit._
+        implicit val s = EnsimeConfigProtocol.parse("")
```

Comment:
why are you doing this? It'll throw an exception. But there is already an implicit EnsimeConfig in scope